test/Microsoft.AspNetCore.SignalR.Client.FunctionalTests/Hubs.cs

```diff
@@ -37,7 +37,20 @@ public class TestHub : Hub
 
         public IEnumerable<string> GetHeaderValues(string[] headerNames)
         {
-            var headers = Context.Connection.GetHttpContext().Request.Headers;
+            var context = Context.Connection.GetHttpContext();
+
+            if (context == null)
+            {
+                throw new InvalidOperationException("Unable to get HttpContext from request.");
```

Comment:
Why would this ever happen in our tests?

Reply:
https://travis-ci.org/aspnet/SignalR/jobs/355210173#L1309

You tell me?! 😕